src/js/post-911-gib-status/components/InfoPair.jsx

```diff
@@ -9,24 +9,41 @@ class InfoPair extends React.Component {
         `usa-grid-full ${this.props.spacingClass}`
         : 'usa-grid-full';
 
-    return (
-      this.props.value &&
-        <div className={gridRowClasses}>
-          <div className="usa-width-one-third">
-            <span><strong>{this.props.label}: </strong></span>
-          </div>
-          <div className="usa-width-one-third">
-            {this.props.value}
-          </div>
+    const row = (
+      <div className={gridRowClasses}>
+        <div className="usa-width-one-third">
+          <span><strong>{this.props.label}: </strong></span>
+        </div>
+        <div className="usa-width-two-thirds">
+          {this.props.value}
         </div>
-    || null);
+      </div>
+    );
+
+    let rowToDisplay;
+    // Some rows should display the value even if the value is 0,
+    // while other rows shouldn't display anything if the value is 0.
+    // If this prop is passed, the value should be displayed regardless
+    // of if it is 0 or not.
+    // If the prop is not passed, check first if the value exists before
```

Comment:
Actually, something like this might be a little clearer:

The displayIfZero prop may be passed in because some rows should display 0 values, while other rows should not display any 0 values. If displayIfZero is false or is not passed in, only display the row if the value is defined and nonzero.